templates/base/.cursor/rules/scaffold-eth.mdc.template.mjs

```diff
@@ -0,0 +1,85 @@
+import { withDefaults } from "../../../utils.js"
+
+const contents = ({ solidityFramework, deployScriptDir }) => {
+  return `---
+description: 
+globs: 
+alwaysApply: true
+---
+
+This codebase contains Scaffold-ETH 2 (SE-2), everything you need to build dApps on Ethereum. Its tech stack is NextJS, RainbowKit, Wagmi and Typescript. Supports Hardhat and Foundry.
+
+It's a yarn monorepo that contains two main packages:
```

Comment:
nit: sometimes there's only nextjs

Reply:
Made it to be "following packages" instead of saying "two" packages fixed at https://github.com/scaffold-eth/create-eth/pull/240/commits/c64e18fb90dbc336985026288ed29b058adfe9f0